README.md

```diff
@@ -12,16 +12,27 @@ pip install git+https://github.com/tataratat/gustaf.git@main
 
 ### Dependencies
 `numpy` is the only strict dependency of `gustaf`.  
-Followings are optional library for more functionalities!
+It can be installed via
 ```
-pip install vedo scipy splinepy
+pip install numpy
 ```
-you could also install dependencies using `conda`. (`splinepy` is only available through pip right now)
+or `conda`.
 ```
-conda install -c anaconda numpy scipy
-conda install -c conda-forge vedo 
+conda install numpy 
 ```
 
+#### Optional Dependencies
+
+1. `splinepy`: All Spline functionalities are dependent on this package. You can install this package via pip `pip install splinepy`.
+2. `vedo`: The main visualization backend. This package can be installed with:
+```
+conda install -c conda-forge vedo
+# or 
+pip install vedo
+```
+If you install `gustaf` from source we recommend to also install `splinepy` from source, see the install instructions for this in the [splinepy docs](tataratat.github.io/splinepy).
+
+
```

Comment:
I just realized that we don't have to separately install numpy, because this is part of `intall_requires`. Can we replace just put a table for dependency?
for example:

| package | pip | conda | description |
| -------- | --- | ---- | ---- |
| ...